docs/usage.rst

```diff
@@ -149,18 +152,34 @@ Many batch systems and e.g. MPI can set a process mask on the application to res
 an application can run. pika will by default take this process mask into account when determining
 how many threads to use for the runtime. ``hwloc-bind`` can also be used to manually set a process
 mask on the application. When a process mask is set, the default behaviour is to use only one thread
-per core in the process mask. Setting ``--pika:threads`` to a number higher than the number of cores
-available in the mask is not allowed. Using ``--pika:threads=all`` will use all the hyperthreads in
-the process mask.
-
-The process mask can explicitly be ignored with the option ``--pika:ignore-process-mask`` or
-overridden with ``--pika:process-mask``. With ``--pika:ignore-process-mask`` pika behaves as if no
-process mask is set. ``--pika:process-mask`` takes an explicit hexadecimal string (beginning with
-``0x``) representing the process mask to use. The mask can also be set with the environment variable
-``PIKA_PROCESS_MASK``. ``--pika:process-mask`` takes precedence over ``PIKA_PROCESS_MASK``.
-``--pika:print-bind`` can be used to verify that the bindings used by pika are correct. Exporting
-the environment variable ``PIKA_PRINT_BIND`` (any value) is equivalent to using the
-``--pika:print-bind`` option.
+per core in the process mask. Setting the number of threads to a number higher than the number of
+cores available in the mask is not allowed. Using ``all`` as the number of threads will use all the
+hyperthreads in the process mask.
+
+The process mask can explicitly be ignored with the environment variable
+``PIKA_IGNORE_PROCESS_MASK=1`` or the command line option ``--pika:ignore-process-mask``. A process
+mask set on the process can explicitly be  overridden with the environment variable
+``PIKA_PROCESS_MASK`` or the command line option ``--pika:process-mask``. When the process mask is
+ignored, pika behaves as if no process mask is set and all cores or hyperthreads can be used by the
+runtime. ``PIKA_PROCESS_MASK`` and ``--pika:process-mask`` take an explicit hexadecimal string
+(beginning with ``0x``) representing the process mask to use. ``--pika:print-bind`` can be used to
```

Comment:
Is it worth linking here to some external documentation giving more details on how to possibly generate such masks etc.?

Reply:
I added links to the man pages of both `hwloc-calc` and `hwloc-distrib`: https://github.com/pika-org/pika/pull/1387/commits/3edf56ee71127286b4cf2cd64a09b77c133c5dbc.